CefSharp/Internals/JavascriptObjectRepository.cs

```diff
@@ -287,6 +305,14 @@ private static JavascriptMethod CreateJavaScriptMethod(MethodInfo methodInfo, bo
             jsMethod.JavascriptName = GetJavascriptName(methodInfo.Name, camelCaseJavascriptNames);
             jsMethod.Function = methodInfo.Invoke;
             jsMethod.ParameterCount = methodInfo.GetParameters().Length;
+            jsMethod.MethodParameters = methodInfo.GetParameters()
+                .Select(t => new MethodParameter()
+                { Name = t.Name,
+                    ParameterType = t.ParameterType,
```

Comment:
Thanks for catching that.  I had planned on implementing those properties, but never did.  I removed those.